.gitignore

```diff
@@ -39,3 +39,6 @@
 
 # debug information files
 *.dwo
+
+# readme
+README.md
```

Comment:
#should not ignore readme file! 😅

Reply:
Not the best practice 😅, because then those wouldn't be tracked by Git and you'd lose any changes made to them 😢.